pkg/skaffold/build/cluster/sources/localdir_test.go

```diff
@@ -101,5 +116,5 @@ func TestPod(t *testing.T) {
 		},
 	}
 
-	testutil.CheckDeepEqual(t, expectedPod, pod)
+	testutil.CheckDeepEqual(t, expectedPod, pod, cmpopts.IgnoreUnexported(resource.Quantity{}))
```

Comment:
We typically use `reflect.DeepEqual` directly when there are unexported fields. Here's an [example](https://github.com/GoogleContainerTools/skaffold/blob/master/pkg/skaffold/build/custom/custom_test.go#L122). This way we won't have to vendor in any new dependencies!

Reply:
Alright. I was wondering why it wasn't vendored in the first place 😄